### ISSUE TITLE
[BUG] Unable to refresh uYou+ out of all apps

### ISSUE THREAD

**Issue by jimmysofat6864** [SideStore/SideStore#472]
### Describe the bug

When I try to refresh uYou+, I get this error message. But when I refresh my other apps, I don't get this error message. The error message is below. I also have a photo for reference as well below. 

`The operation couldn’t be completed. Unable to install the app: Failed to load Info.plist from bundle at path /private/var/containers/Bundle/Application/0D048302-4D9F-4C69-8F43-6B23225739DF/App.app/PlugIns/MessagesExtension.appex; Extra info about "/private/var/containers/Bundle/Application/0D048302-4D9F-4C69-8F43-6B23225739DF/App.app/PlugIns/MessagesExtension.appex/Info.plist": Couldn't stat /private/var/containers/Bundle/Application/0D048302-4D9F-4C69-8F43-6B23225739DF/App.app/PlugIns/MessagesExtension.appex/Info.plist: No such file or directory`
![IMG_D557495276D9-1](https://github.com/SideStore/SideStore/assets/69532417/80f44425-a3f7-4fb5-9793-f7a7c83e2a66)

### Instructions to reproduce

1. Install SideStore
2. Attempt to install the aforementioned uYou+ IPA linked [here.](https://github.com/qnblackcat/uYouPlus/releases/tag/v18.14.1-3.0)
3. Error message shown 

### What version of SideStore are you using?

0.5.1

### Other info

Seems to only affect uYou+ as my other apps work fine. Also I am on iOS 17 if that changes anything.

**Comment by jimmysofat6864** [comment]
Also worth noting that on 0.5.0 and iOS 16 I wouldn't have this issue. I would keep getting staging directory but if I press refresh enough times it would go through successfully and install correctly.

**Comment by naturecodevoid** [comment]
See uYou+'s FAQ.

TLDR: You need to do a clean install of uYou+.

This should be possible for us to fix so I'll leave the issue open, but it is not a priority at all.

**Comment by OniriCorpe** [comment]
This bug also blocks the deactivation feature for uYou+

**Comment by voidxO7** [comment]
uYou+ is now outdated I think. Use uYou+Extra instead.

**Comment by Spidy123222** [comment]
This was resolved
